templates/versafix-1/template-versafix-1.html

```diff
@@ -341,7 +356,7 @@
         frameTheme { label: Frame Style; }
         frameTheme:preview { -ko-background-color: @[backgroundColor] }
 
-        footerBlock { label: Footer Block; theme: contentTheme }
+        footerBlock { label: Unsubscribe Block; theme: frameTheme }
```

Comment:
no, we don't need to follow how it was, frameTheme is not flexible enough for blocks, all blocks have to have content theme